python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -288,6 +288,54 @@ def test_axes(data):
         assert_eq(e, a)
 
 
+def test_dataframe_truncate_axis_0():
+    cdf1 = cudf.DataFrame(
```

Comment:
It appears these tests only deal with one dataframe. Can we avoid the suffix `1` on variables? Our tests sometimes use the names `gdf` (for GPU DataFrame) and `pdf` (for Pandas DataFrame).